modules/ROOT/pages/tuning-test-validations.adoc

```diff
@@ -0,0 +1,85 @@
+= Performance Test Validations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To start validating your application's performance, first understand the prerequisites to testing, follow the recommended best practices, and then execute performance testing.
+
+
+== Prerequisites to Perform Test Validations
+
+Before testing the performance, consider the following points:
+
+* Confirm that your Mule app and its functions work as expected because a wrong flow can give false-positive data.
+* Establish the performance criteria for the test cases to determine the goal and scope for tuning. The following questions can help you clarify the performance requirements:
+** What are the expected average and peak workloads?
+** Is the use case emphasis on throughput, concurrency, response time or size of messages? Some common cases are:
+*** Focus on throughput when handling a large volume of transactions is a high priority.
+*** Focus on response time or latency if spikes affect user experience and becomes frustrated.
+*** Focus on concurrency if it is necessary to support a high number of users connecting at the same time.
+*** Focus on managing large messages when the application is transferring, caching, storing, and or processing a payload bigger than 1 MB.
+** What is the minimum acceptable throughput?
+** What is the maximum acceptable response time?
+
+
+== Performance Test Validations Best Practices
+To apply performance testing consider the following best practices.
+
+=== Use a Controlled Environment
+Use a self-controlled environment to obtain repeatable, reliable and consistent data.
+To control and stabilize the environment:
+
+* Use an infrastructure made by a triplet:
+** Use a dedicated host for the Mule application to have isolated information preventing other running processes from interfering with the results in case of on-premises tests.
```

Comment:
Do we really need to explain the reasons for the dedicated hosts? I suggest...:
```suggestion
** Use separate, dedicated hosts for the Mule application, for running load client tools, and for running backend services.
```
OR
```suggestion
** Use a triplet of separate, dedicated hosts for the Mule application, the load client tools, and the backend services.
```

Reply:
If you want to use the separate bullets, then...:
```suggestion
** Use a dedicated host for the Mule application to isolate app information, preventing other running processes from interfering with the results of on-premises tests.
```